core/templates/pages/story-editor-page/editor-tab/story-node-editor.component.ts

```diff
@@ -57,6 +57,8 @@ export class StoryNodeEditorComponent implements OnInit, OnDestroy {
   @Input() acquiredSkillIds: string[];
   @Input() plannedPublicationDateMsecs: number;
 
+  MAX_CHARS_IN_EXPLORATION_TITLE = 36; 
```

Comment:
We already store these values as a constants in constants.ts, why are they being defined here? This doesn't seem like the correct fix